### ISSUE TITLE
2.5.0 Build fail on i686 arch

### ISSUE THREAD

**Issue by Pro-pra** [falkTX/Carla#1671]
```
make[1]: Leaving directory '/builddir/build/BUILD/Carla-2.5.0/source/modules/ysfx'
sources/ysfx_utils.cpp:40:29: error: static assertion failed: 64-bit large file support is not enabled
   40 | static_assert(sizeof(off_t) == 8, "64-bit large file support is not enabled");
      |               ~~~~~~~~~~~~~~^~~~
make[1]: *** [Makefile:175: ../../../build/ysfx/Release/sources/ysfx_utils.cpp.o] Error 1
```

Compile flags:
-Wall -Wextra -pipe -DBUILDING_CARLA -DREAL_BUILD -MD -MP -fno-common -fPIC -DPIC -DNDEBUG -O3 -ffast-math -fdata-sections -ffunction-sections -mtune=generic -msse -msse2 -mfpmath=sse -fvisibility=hidden -fno-gnu-unique -DHAVE_DGL -DHAVE_OPENGL -DDGL_OPENGL -DDONT_SET_USING_DGL_NAMESPACE -DDGL_NAMESPACE=CarlaDGL -DDGL_FILE_BROWSER_DISABLED -DDGL_NO_SHARED_RESOURCES -DHAVE_FLUIDSYNTH -DHAVE_FLUIDSYNTH_INSTPATCH -DHAVE_HYLIA -DHAVE_JACK -DHAVE_LIBLO -DHAVE_LIBMAGIC -DHAVE_PYQT -DHAVE_SDL -DHAVE_SDL2 -DHAVE_SNDFILE -DHAVE_X11 -DUSING_JUCE -DJUCE_APP_CONFIG_HEADER='"AppConfig.h"' -DUSING_RTAUDIO -DCARLA_LIB_EXT=\".so\" -std=gnu++11 -O2 -fomit-frame-pointer -gdwarf-4 -Wstrict-aliasing=2 -pipe -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2 -fPIC -fstack-protector-strong --param=ssp-buffer-size=4 -m32 -march=i686 -mtune=generic -msse -mfpmath=sse -fasynchronous-unwind-tables -mstackrealign -fvisibility-inlines-hidden -fvisibility-inlines-hidden

May be its warning and need remove -Wall ?

**Comment by falkTX** [comment]
This was fixed in a commit after 2.5.0 release.
I am yet to do a 2.5.1 bugfix

**Comment by Pro-pra** [comment]
Thank you!
